content/en/docs/contributing/pr-checks.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Pull Request Checks
+description: Learn how to make your Pull Request successfully pass all checks
+weight: 40
+cSpell:ignore: REFCACHE
+---
+
+When you raise a
+[Pull Request](https://docs.github.com/en/get-started/learning-about-github/github-glossary#pull-request)
+with the
+[opentelemetry.io repository](https://github.com/open-telemetry/opentelemetry.io)
+a set of checks will be run. Those checks will verify, …
+
+- … if you have signed the [CLA](#easy-cla).
+- …if your commit can be deployed via [Netlify](#netlify-deployment)
+  successfully.
+- … if your changes are in line with some [style guidelines](#style-checks).
+
+{{% alert title="Note" color="primary" %}}
+
+If any of your PR checks fails, try to
+[fix content issues automatically](/docs/contributing/#fix-content-issues-automatically)
+first by running `npm run test-and-fix` locally.
+
+Additionally, you can comment `/fix:all` on your Pull Request. This will make
+the OpenTelemetry Bot run those commands on your behalf and update your PR. Make
+sure that you pull those changes locally.
+
+{{% /alert %}}
+
+## Easy CLA
+
+This check will fail, if you didn't
+[sign the CLA](/docs/contributing/#sign-the-cla).
```

Comment:
```suggestion
This check will fail, if you didn't
[sign the CLA](/docs/contributing/#sign-the-cla).
```
```suggestion
This check fails if you haven't
[signed the CLA](/docs/contributing/#sign-the-cla).
```

Reply:
Can we add a link to the CLA instructions?

Reply:
This links back to the contributing index page where it is outlined what needs to be done to sign the CLA